tests/src/Common/CoreFX/CoreFX.depproj

```diff
@@ -10,8 +10,8 @@
       <CLRTestKind>SharedLibrary</CLRTestKind>
       <IsTestProject>false</IsTestProject>
       <!-- System.Composition and System.Composition.AttributedModel use different versioning conventions -->
-      <SystemCompositionVersions>1.3.0-preview3-26501-04</SystemCompositionVersions>
-      <XUnitNetcoreExtensionsVersion>2.2.0-preview1-02902-01</XUnitNetcoreExtensionsVersion>
+      <SystemCompositionVersions>1.3.0-$(MicrosoftPrivateCoreFxNETCoreAppPackageVersion.Substring(6))</SystemCompositionVersions>
+      <XUnitNetcoreExtensionsVersion>2.2.0-preview1-03025-01</XUnitNetcoreExtensionsVersion>
```

Comment:
Composition packages version has been updated to always point to latest v1.3.0 available. Version `1.3.0-preview3-26501-04` has been always chosen over any `1.3.0-preview1-xxxxx-xx`  see https://github.com/dotnet/corefx/issues/31395